packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -160,6 +159,18 @@ SearchSelectInput.propTypes = {
    * Determines if option groups will be separated by a divider
    */
   showOptionGroupDivider: PropTypes.bool,
+  /**
+   * The default set of options to show before the user starts searching. When set to `true`, the results for `loadOptions('')` will be autoloaded.
+   */
+  defaultOptions: PropTypes.oneOfType([
+    PropTypes.bool,
+    PropTypes.arrayOf(
+      PropTypes.shape({
+        value: PropTypes.string.isRequired,
+        label: PropTypes.string.isRequired,
```

Comment:
I meant if it's optional, keep it optional but just include the prop in the object, so it's also documented in the readme.

```suggestion
        label: PropTypes.string,
```